demo/docs/features/metas.md

```diff
@@ -0,0 +1,43 @@
+# Metas
+
+Metas can be seen as labels. They are additional data that accompany the errors, events etc. in order to filter them
+better.
+
+## Description
+
+### Client
+
+Grafana JavaScript Agent requires `app` meta as the only mandatory meta when initializing the library. Besides `app`,
```

Comment:
I avoided the `name` here since we also send the `version` and `environment`. My intention was to make the user curious to look into the code and check what it contains and in the actual docs of the library to specify exactly what's required and what not.